core/src/test/java/org/infinispan/jmx/CacheLoaderAndCacheWriterInterceptorMBeanTest.java

```diff
@@ -29,39 +31,48 @@
 public class CacheLoaderAndCacheWriterInterceptorMBeanTest extends SingleCacheManagerTest {
    private ObjectName loaderInterceptorObjName;
    private ObjectName storeInterceptorObjName;
-   private MBeanServer threadMBeanServer;
    private AdvancedLoadWriteStore store;
    private static final String JMX_DOMAIN = CacheLoaderAndCacheWriterInterceptorMBeanTest.class.getName();
+   private final MBeanServerLookup mBeanServerLookup = MBeanServerLookupProvider.create();
 
    @Override
    protected EmbeddedCacheManager createCacheManager() throws Exception {
-      cacheManager = TestCacheManagerFactory.createCacheManagerEnforceJmxDomain(JMX_DOMAIN);
       ConfigurationBuilder configuration = getDefaultStandaloneCacheConfig(false);
       configuration
-         .jmxStatistics().enable()
-         .persistence()
+            .jmxStatistics().enable()
+            .persistence()
             .passivation(false)
             .addStore(DummyInMemoryStoreConfigurationBuilder.class);
 
+      GlobalConfigurationBuilder globalConfiguration = new GlobalConfigurationBuilder();
+      globalConfiguration
+            .globalJmxStatistics()
+            .allowDuplicateDomains(false)
+            .jmxDomain(JMX_DOMAIN)
+            .mBeanServerLookup(mBeanServerLookup)
+            .enabled(true);
```

Comment:
`cacheContainer().statistics(exposeGlobalJmx)`?

Reply:
done. all of those